### PR TITLE
MolNet Jenkins build

### PR DESCRIPTION
**Pull request by miaecle**
This PR adds in:

- An update of weekly molnet benchmark run on jenkins

- several bugs on past tensorflow models(squeeze operation error due to appearance of batch size of 1)

- Benchmark script to generate learning curve(performance versus training set size) as suggested by MolNetv3

### CODE REVIEW

File: devtools/jenkins/jenkins.sh [hunk 1]
```diff
@@ -6,13 +6,22 @@ source activate $envname
 python setup.py install
 
 rm examples/results.csv || true
+export DEEPCHEM_DATA_DIR='/tmp/molnet_'$envname
+mkdir $DEEPCHEM_DATA_DIR
 cd examples
-python benchmark.py -d tox21
-export retval1=$?
+python benchmark.py -d tox21 -m weave -m graphconv -m tf_robust -m tf -m irv -m xgb -m logreg --seed 123
```
Comment (rbharath) [inline devtools/jenkins/jenkins.sh]: How long do the benchmarks take to run in total?

**Comment by lilleswing** [comment]
We can run weekly -- were running the full benchmark weekly before.

File: devtools/jenkins/jenkins.sh (hunk 1 above)
Comment (miaecle) [reply devtools/jenkins/jenkins.sh]: 4 to 5 hours at this setting

**Comment by miaecle** [comment]
@rbharath @lilleswing Other than PCBA and QM9 we should be fine to tun all benchmarks. Also we might want to skip kernel svm, random forest and xgboost since they take a long time and are pretty stable.

**Comment by rbharath** [comment]
LGTM

**Pull request merged**